requirements_test.txt

```diff
@@ -1,8 +1,8 @@
 # requirements for running tests
 # Expects some apt packages are already installed.
-# see .travis.yml for more details.
+# see .github/workflows/python_tests.yml for more details.
 
--r requirements_common.txt
+-r requirements.txt
 debugpy>=1.2.0
```

Comment:
[`debugpy`](https://github.com/microsoft/debugpy) is not required in our GitHub Actions but we need it to be in the Docker container if we want to https://github.com/internetarchive/openlibrary/wiki/Debugging

Reply:
Right, so this should be moved to `requirements.txt` instead.